### ISSUE TITLE
deno 1.14.0 import puppeteer does proc.kill(9)

### ISSUE THREAD

**Issue by jimmont** [lucacasonato/deno-puppeteer#32]
reproduce with `deno upgrade` (or explicitly `deno upgrade --version 1.14.0` then create file.js with:
```js
import puppeteer from 'https://deno.land/x/puppeteer/mod.ts';
```

then `deno run -A --unstable file.js`

resulting error 
```
error: TS2345 [ERROR]: Argument of type 'number' is not assignable to parameter of type 'string'.
        this.proc.kill(9);
                       ^
    at https://deno.land/x/puppeteer@5.5.1/vendor/puppeteer-core/puppeteer/deno/BrowserRunner.ts:125:24
```

environment: macos 11.6 M1 2020; deno release in all instances mentioned here are for aarch64-apple-darwin
temporary solution: `deno upgrade --version 1.13.2`

**Comment by psidex** [comment]
Can confirm this is also happening for me on Windows running Deno 1.14.0

**Comment by lucacasonato** [comment]
The fix will have to wait for Deno 1.14.1, as there is a bug in 1.14.0. https://github.com/denoland/deno/issues/12133

**Comment by sandrolain** [comment]
Hi, I am having the same problem with Deno 1.14.1 and with 1.14.2 released today, using deno-puppeteer 9.0.1
